operations/node_management/Add_Remove_Replace_NCNs/Update_NCN_BIOS_TPM_State.md

```diff
@@ -0,0 +1,86 @@
+# Update NCN BIOS TPM State
+
+## Description
+
+Enable Trusted Platform Module (TPM) in the BIOS on the new NCN if TPM is being used on the other NCNs.
+**Skip this step if TPM is not being used** and proceed to the next step to [Boot NCN and Configure](Boot_NCN.md).
```

Comment:
`**Skip this step if TPM is not being used**` can this be better defined? As an admin following this procedure how do I know if the TPM is not being used. Checking the TPM state of the other NCNs could be used as an indicator if this procedure needs to be followed.